packages/backend/src/nest/common/utils.ts

```diff
@@ -233,3 +238,21 @@ export async function createPeerId(): Promise<PeerId> {
   const peerId = await PeerId.create()
   return peerIdFromKeys(peerId.marshalPubKey(), peerId.marshalPrivKey())
 }
+
+const KEY_LENGTH = 32
+
+export function generateLibp2pPSK(bytes: Uint8Array | NodeJS.WriteStream, key?: Uint8Array) {
```

Comment:
I can move it there but then I'll make it a static method because we use it in tests and initializing whole libp2p service just to generate psk is an overkill.